bitnami/apache/values.yaml

```diff
@@ -139,6 +139,7 @@ extraPodSpec: {}
 ## @param cloneHtdocsFromGit.enabled Get the server static content from a git repository
 ## @param cloneHtdocsFromGit.repository Repository to clone static content from
 ## @param cloneHtdocsFromGit.branch Branch inside the git repository
+## @param cloneHtdocsFromGit.enableAutoRefresh Enables an automatic git pull with a sidecar container
```

Comment:
We need to update the README.md to reflect this new parameter you're introducing. To do so, please use the [readme-generator](https://github.com/bitnami-labs/readme-generator-for-helm) tool that automatically updates the README.md for you based in this metadata.

Reply:
hope this is fixed now. was under the impression the readme update is handled by the build pipeline.